.github/workflows/tests.yaml

```diff
@@ -1259,6 +1259,21 @@ jobs:
         run: |
           cd apps/upgrade-journey-raft
           ./scripts/metadata_only_voters_check.sh
+  multivector-version-compatibility:
```

Comment:
Can you add needs and if checks to your pipeline in order to make your pipeline running with a certain Weaviate version and also to add the ability to run this one pipeline only if needed.
I'm pasting an example of missing settings:
```
needs: [newer-or-equal-than-1_25]
    if: ${{ (needs.newer-or-equal-than-1_25.outputs.check == 'true') && ( github.event.inputs.test_to_run == 'upgrade-journey-raft-metadata-only-voters' || github.event.inputs.test_to_run == '') }}
```